specifications/M1/identity.protocol.base.md

```diff
@@ -109,3 +110,14 @@ scope of this specification.
 The Self-Issued ID Token request MAY contain the `bearer_access_scope` authorization request parameter which is set to a
 list of space-delimited scopes the response `VP Access Token` set in the `access_token` claim will be enabled for. If
 no `bearer_access_scope` parameter is present, the `access_token` claim MUST not be included.
+
+# 7. The Identity and Trust Protocol Context
+
+The `Json-ld context` URI for the all identity and trust specifications is:
+
+`https://w3id.org/tractusx-trust/v[version]`
```

Comment:
Does it make to use `iatp` also for the namespace and context urls?

Reply:
That is optional for spec implementors. In the spec, yes we should use the same prefix, but this section is only about what implementations must do, not the spec iteself.